docs/data.md

```diff
@@ -54,9 +57,28 @@ Metadata columns
 18. url # url to the publication
 ```
 
+## Inconsistencies
+
+Paper id | Title in metadata | Title in publication
+-------- | ----------------- | ---------------------
+0b4b4e5bb8d0d5167eec1e203b5dad283bd364a5 | Doubling healthy lifespan using drug synergy | 2
+60abca9911a64805e51aa8deb70bb238c2f3d414 | Influenza-Associated Mortality in Georgia (2009–2011) | )
+05c7871a5da795da7c03748bb7eff551bc66b721 | " " | A
+54521bd72c1242be4cc154f66ebaf5759478e4cf | How Should U.S. Hospitals Prepare for Coronavirus Disease 2019 (COVID-19)? | \
+31ff32abd2da30cf9a0ed4414fb83e19eb874314 | Truth in the Details | \
+97b02266fec0d45796c970161d3ebb4ac56b8f6f | Summary of Suggestions From the Task Force for Mass Critical Care Summit, January 26–27, 2007 | *
+
+## Duplicates
+
+There are 269 rows that mention the same sha. There is not duplicate pmcid.
```

Comment:
didnt have time to actually check this, so you may have accounted for this/it's irrelevant, but FYI that from the metadata.csv file you have to use different things as the unique ID. I found 
Of the 29500 publications:
pmid is given for 16730 (57%) of them
of those without pmids, the doi is given for 9950 (34%) of them
and of those without pmids or dois, the pmcid is given for 2767 (9%) of them
 and 53 (0%) have no pmid, doi or pmcid
https://github.com/wellcometrust/datalabs/blob/master/analysis/covid_scientific_evidence/Liz%20-%20Get%20funding%20data.ipynb

(ignore if this is irrelevant!)

Reply:
there is `cord_uid` as a persistent unique identifier but i guess you mean a different id links to the files each time. i thought for the files you either use sha or pmcid.